hack/usage/wait-for.sh

```diff
@@ -47,10 +47,10 @@ retries=0
 while [ "${retries}" -lt "${TIMEOUT}" ]; do
   if [ -z "$NAMESPACE" ]; then
     # Get the condition types in jsonpath format and pipe to yq to extract the value of conditions
-    CONDITION_STATES=$(kubectl get "${RESOURCE_TYPE}" "${OBJECT_NAME}" --request-timeout='1s' -o json | yq '.status.conditions')
+    CONDITION_STATES=$(kubectl get "${RESOURCE_TYPE}" "${OBJECT_NAME}" --request-timeout='1s' -o json | yq '.status.conditions') || true
```

Comment:
I guess I missed this early, but why are we setting `--request-timeout=1s` explicitly in these calls here?

Reply:
Fixed